code/modules/jobs/job_types/vtcc.dm

```diff
@@ -622,6 +622,7 @@
 	spawn_positions = 1
 	supervisors = "the Merchant & the Shopkeeper."
 	description = "Operate the shop when the Merchant's asleep or just isn't there, push and pull those crates to and fro like your life depends on it (and when your paycheck is based on performance, well, it certainly does). If there are no crates to pick up, head out into the wastes and start scavenging for anything you could sell at an extortionate price."
```

Comment:
this kind of suggests they go scav boss hunting despite your new forbids text though, may want to clarify it like 

``` suggestion
	description = "Operate the shop when the Merchant's asleep or just isn't there, push and pull those crates to and fro like your life depends on it (and when your paycheck is based on performance, well, it certainly does). If there are no crates to pick up, head out into the wastes and start scavenging for anything you could sell at an extortionate price. Just be sure to leave the stock alone. Shop inventory is for sale ONLY, and should not be taken for personal use."
```

Reply:
Fair point!